__tests__/app.test.js

```diff
@@ -76,28 +76,11 @@ describe("app", () => {
                 topic: expect.any(String),
                 created_at: expect.any(String),
                 votes: expect.any(Number),
+                comment_count: "11",
               })
             );
           });
       });
```

Comment:
Can't see the below error handling tests replaced anywhere - did you delete it completely from the test suite?